assets/css/custom.css

```diff
@@ -0,0 +1,18 @@
+.side-nav-li{
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	word-break: break-all;
+}
+.side-nav-li:hover{
```

Comment:
@dheeraj135 It would be great if you could shed some light on why do we need additional CSS here which overwrites the default one.

Reply:
By default, entire title of page is displayed in the navigation bar. This css is added to hide the overflow and display entire text on hover.

Reply:
Is this issue resolved? @sahilarora535